client/src/components/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { FaBeer } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    console.log('click');
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="parent">
+          <div className="shareAndSaveContainer">
+            <div className="shareButton" onClick={this.handleClick}>
+              <a className="share" href="#"><FaBeer />  Share</a>
```

Comment:
Will this link actually navigate the user to another page? If not, let's not make it an `<a>` tag. Other options include:

- `<div>`
- `<span>`
- `<i>` (Icon)